components/WelcomeScreen.tsx

```diff
@@ -3,29 +3,31 @@ import { Button, Jumbotron } from 'react-bootstrap'
 
 import { HandleSignup } from '../types/user'
 
-const WelcomeScreen = ({
-  handleSignup
-}: {
+type Props = {
   handleSignup: HandleSignup
-}): JSX.Element => (
+}
+
+const WelcomeScreen = ({ handleSignup }: Props): JSX.Element => (
   <>
     <Jumbotron>
       <h1>Welcome to the {process.env.SITE_TITLE}!</h1>
       <p>
         Here, you can view documentation for and gain access to the{' '}
         {process.env.API_NAME}.
       </p>
-      <p>
-        <Button className="mr-3" onClick={handleSignup} variant="primary">
-          Sign up for API access
-        </Button>
-        <Button
-          href={process.env.API_DOCUMENTATION_URL}
-          variant="outline-primary"
-        >
-          View API documentation
-        </Button>
-      </p>
+      {process.env.API_MANAGER_ENABLED && (
```

Comment:
Also include checking `API_DOCUMENTATION_URL` in the condition (see my comment on the welcome page).

Reply:
Done in 3a29c33ffe0b1276cc11e44d7ce6ffe766f99d71